modules/core/module.go

```diff
@@ -117,21 +117,11 @@ func (am AppModule) RegisterInvariants(ir sdk.InvariantRegistry) {
 	// TODO:
 }
 
-// Route returns the message routing key for the ibc module.
-func (am AppModule) Route() sdk.Route {
-	return sdk.Route{}
-}
-
 // QuerierRoute returns the ibc module's querier route name.
 func (AppModule) QuerierRoute() string {
 	return host.QuerierRoute
```

Comment:
Is this function still needed?

Reply:
we can remove. Haven't had a moment this week to get back to this pr